pom.xml

```diff
@@ -62,6 +62,23 @@
       </dependency>
 
       <!-- Jenkins modules -->
+      <dependency>
+          <groupId>io.jenkins.tools.bom</groupId>
+          <artifactId>bom-2.332.x</artifactId>
+          <version>1181.v04b_21d4b_0d6c</version>
+          <scope>import</scope>
+          <type>pom</type>
+      </dependency>
```

Comment:
Make the BOM entry the first in the list of dependencies, so it stands out visually.

Reply:
Taken care of this. Thank you